logic/consumer-proguard-rules.pro

```diff
@@ -25,5 +25,20 @@
    <init>(...);
 }
 
+-keep class net.sqlcipher.** {
+    *;
+}
+
+-keep  class net.sqlcipher.database.** {
+    *;
+}
+-keep class net.zetetic.** {
+    *;
+}
+
+-keep  class net.zetetic.database.** {
+    *;
+}
+
```

Comment:
Unnecessary to put twice, isn't it?

If `persistence` has it, it should pass forward 🤔

Reply:
i will remove it from logic, but does that mean the app does not need the same rules ?

Reply:
The idea of the `consumerProguardFiles` is that they are bundled all the way until up to the application.

The application, if being minified, will merge all the proguard files together.

In this case:
`persistence` -> `logic` -> `Android app`.

The Android App should see the Proguard rules bundled within `persistence`.

Just like it should be able to see Proguard rules bundled within, let's say... OkHttp or Ktor.

Ktor -> `network` -> `logic` -> `Android App`.